### PR TITLE
fix(reactmultiemail): account for emails being entered as "Name <name…

### PR DESCRIPTION
**Pull request by jhaltercmu2020**
…@domain.com>"

account for emails being entered as "Name <name@domain.com>". The reason why it is helpful to be able to enter emails in this format is when copying a list of emails from Google or most email providers, the format is "Name <email@domain.com>"

### CODE REVIEW

File: react-multi-email/ReactMultiEmail.tsx [hunk 1]
```diff
@@ -66,7 +66,8 @@ export function ReactMultiEmail(props: IReactMultiEmailProps) {
     async (value: string, isEnter?: boolean) => {
       const validEmails: string[] = [];
       let inputValue: string = '';
-      const re = new RegExp(delimiter, 'g');
+      const isNameFormat = value.includes("<") && value.includes(">") ? true : false;
```
Comment (thomasJang) [inline react-multi-email/ReactMultiEmail.tsx:69]: I think there's a simple way to make the code like below
```
const isNameFormat = value.includes('<') && value.includes('>');
```

**Review by thomasJang — CHANGES_REQUESTED** [review]
image
If you enter the value and enter the Enter key as shown in the picture, it doesn't work.

File: react-multi-email/ReactMultiEmail.tsx (hunk 1 above)
Comment (jhaltercmu2020) [reply react-multi-email/ReactMultiEmail.tsx:69]: @thomasJang This is much simpler. I will make this change.

**Comment by jhaltercmu2020** [comment]
> image
> If you enter the value and enter the Enter key as shown in the picture, it doesn't work.

@thomasJang Copy and Paste works but something is definitely broken when trying to type emails in that format. I will fix it and let you know when I have it working so you can try again. Thank you for reviewing!

**Comment by thomasJang** [comment]
@jhaltercmu2020 

Thank you for your effort.
And if possible, I think it would be good to use regular expressions to check if 'value' is an email pattern.
And furthermore, I thought it would be better to provide email patterns as an option for users to choose from.

**Comment by jhaltercmu2020** [comment]
@thomasJang I made some updates based on your comments. Can you take a look? Thank you!

**Review by thomasJang — APPROVED** [review]
Thank you !

**Comment by jhaltercmu2020** [comment]
@thomasJang Would you be able to tag 2 other colleagues to review this?

**Review by vinyl810 — APPROVED** [review]
Sorry for the late review. Looks good to me.

**Review by hwonda — APPROVED** [review]
thank you for your contribution. Looks good to me.

**Comment by jhaltercmu2020** [comment]
@hwonda @thomasJang When is the next release scheduled to happen? We are wondering if this can get deployed so we can install the new dependency in our project?

**Comment by thomasJang** [comment]
It 1.0.17 is just out, sorry for late, Thank you for your effort.

**Pull request merged**